exercise3.js

```diff
@@ -0,0 +1,122 @@
+// Task 1. Add new Products: Google Pixel 6 Pro, Motorola Moto G31, Honor X7
+
+ products.push('Google Pixel 6 Pro', 'Motorola Moto G31', 'Honor X7')
+console.log(products)
+/*PS C:\Users\bisrat\Desktop\javascript1> node exercise3.js
+hello
+[
+  'iPhone 13',          'iPhone 13 Pro',
+  'iPhone 13 Pro Max',  'iPhone 12',
+  'iPhone 12 Pro',      'OnePlus 10 Pro',
+  'OnePlus 9',          'Nokia G20',
+  'Samsung Galaxy S20', 'Samsung Galaxy S21',
+  'Samsung Galaxy S22', 'Sony Xperia 10',
+  'Xiaomi Redmi 9A',    'Google Pixel 6 Pro',
+  'Motorola Moto G31',  'Honor X7'
+]
+
+ Task 2. remove Sony Xperia 10 from the list */
+
+products.splice(11,1)

```

Comment:
Task 2 needs more work.
- Replace this splice with filter. Filter out "Sony Xperia 10".
- delete these printouts for easier readability.